go.mod

```diff
@@ -78,5 +78,3 @@ replace (
 	k8s.io/client-go => github.com/openshift/kubernetes-client-go v0.0.0-20210826123502-7208c21f5119
 	k8s.io/kubectl => github.com/openshift/kubernetes-kubectl v0.0.0-20210730111826-9c6734b9d97d
 )
-
-replace github.com/mdlayher/vsock => github.com/cfergeau/vsock v0.0.0-20210707100525-4def293f672e
```

Comment:
The addition of `github.com/mdlayher/vsock v1.1.0` in go.mod was done in the commit updating `gvisor-tap-vsock`, but this first commit is not buildable without this change. I'd squash both commits, mentioning in the commit log that we can drop our fork of mdlayher/vsock since it has the API we need in the version we are switching to.
I've tested vsock socket activation of the crc daemon with v1.1.0 and it works as expected.